tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -557,7 +557,7 @@ void DxcContext::ExtractRootSignature(IDxcBlob *pBlob, IDxcBlob **ppResult) {
   hlsl::InitDxilContainer(&newHeader, 1, containerSize); 
   CComPtr<IMalloc> pMalloc;
   CComPtr<hlsl::AbstractMemoryStream> pMemoryStream;
-  IFT(CoGetMalloc(1, &pMalloc));
+  IFT(DxcCoGetMalloc(1, &pMalloc));
```

Comment:
I think we can get rid of this, and just use `DxcGetThreadMallocNoRef()` in place of pMalloc below.

Reply:
Done.